http/vibe/http/common.d

```diff
@@ -638,6 +668,11 @@ final class Cookie {
 		none = raw
 	}
 
+	/// Cookie anme
+	@property void name(string name) { m_name = name; }
+	/// ditto
+	@property string name() const { return m_name; }
```

Comment:
I don't really like the data redundancy that this introduces. For now it would really be better to keep the API as it is and let `parseHTTPCookie` return a pair of name and cookie. It should also be marked `package`, so that we can discuss the ideal API in a separate session, no need to conflate this with the functional changes of this PR.